src/Core/Geographic/Coordinates.js

```diff
@@ -224,8 +224,32 @@ export function convertValueToUnit(unitIn, unitOut, value) {
 function Coordinates(crs, ...coordinates) {
     this._values = new Float64Array(3);
     this.set(crs, ...coordinates);
+
+    Object.defineProperty(this, 'surfaceNormal',
+        {
+            configurable: true,
+            get: () => {
+                const result = this._normal || this.computeSurfaceNormal();
```

Comment:
And as you use a internal member (`_normal`) to store the value, you can also copy the normal when it makes sense (ie : when going from 4326 to 4978 and the otherway around).

So add something similar to what you did here: https://github.com/iTowns/itowns/pull/494/files#diff-7c37ee347b0423d12e6624421e03f4caR142

```js
target._normal = coordsIn.surfaceNormal;
```

and you can then close PR #494 :)

Reply:
And I still prefer the `geodesicNormal` or `geodesicSurfaceNormal` terms because they contain an important piece of information: this normal isn't computed using a DEM/MNT.